crates/server/src/graphql/modules/user/query/users.rs

```diff
@@ -0,0 +1,84 @@
+use std::str::FromStr;
+
+use async_graphql::connection::{query, Connection, Edge, EmptyFields};
+use async_graphql::{Context, Result};
+use gabble::shared::pagination::Pagination;
+use gabble::user::model::{Email, Username};
+use gabble::user::repository::UserFilter;
+use pxid::graphql::Pxid;
+
+use crate::context::SharedContext;
+use crate::graphql::connection_details::ConnectionDetails;
+use crate::graphql::modules::user::types::User;
+
+pub type UsersConnection = Connection<Pxid, Users, ConnectionDetails, EmptyFields>;
```

Comment:
Here you want to use: `User` from types over `Users` GraphQL Query.

```suggestion
pub type UsersConnection = Connection<Pxid, User, ConnectionDetails, EmptyFields>;
```